lib/checks/navigation/identical-links-same-purpose-after.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * Skip, as no results to curate
+ */
+if (results.length < 2) {
+	return results;
+}
+
+/**
+ * Filter results for which `result` is undefined & thus `data`, `relatedNodes` are undefined
+ */
+const filteredResults = results.filter(
+	({ result, data }) => result !== undefined && !!data
```

Comment:
```suggestion
	({ result, data }) => result !== undefined
```

That second bit does nothing